### PR TITLE
🌐 Add German translation for `docs/de/docs/tutorial/query-params.md`

### PR DESCRIPTION
**Pull request by nilslindemann**
← `tutorial/path-params.md` (#10290)
→ `tutorial/body.md` (#10295)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1aefec0e95b274201e74483e1e864c8a877b697e at: https://f4500ac0.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 54a57a5fa54cf8e79f717b460e92440e4758f335 at: https://d7b2ac38.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4578e891d5ac139538233c619a9e37cbf15e1e1b at: https://9e8ede91.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f3a5a4a79bbd23b974db8fb69b9600547ea0fc6c at: https://de94d7fc.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 54462b741a53d3e44cafbf7c536cbc03926cf425 at: https://f5c96e8b.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 568187be80153a6bc492c6d5f538d7417523ec79 at: https://006ab5b2.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 43b47b5a36a96b2bc055e31109a6b2ba645825ed at: https://13437bc9.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3e8b4bfe5627a1710dc802c3c2cf777774d92729 at: https://bdcb6afa.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9b06b80991a430522bbcccfcf4940acd7ff7b969 at: https://eb72d344.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit d197c7a6b9e1732e3c71859915b1985882c388b6 at: https://0648007c.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit d45aa11e192bf8d7a3beff29e2b7cd4f852499a4 at: https://a10ed620.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8be6d7cd39b1c26a352e3af0827a32ec37da6aa5 at: https://514e49e5.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6a3a4a3aafdc834df6e4d7e7994fb293628f12d8 at: https://a2114f28.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 26e0009a2944c88a724c9797316ce3f5c7a13d03 at: https://f2a87c15.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f4c6e87f357a9437f7c884dd966b83be83f075f7 at: https://e2c863a0.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6b92c0afb93747253b5fb17af86e3de9e2da26a3 at: https://cf5e441b.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3dbe4555e86a06ce375bad8c74b9ebdb87a500c3 at: https://02e21f12.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit c150a810a4beee856b91ae0081c9782999e7e528 at: https://6c520f83.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit be853cff880cb408b18606d5246b156400fae661 at: https://ba2f88bc.fastapitiangolo.pages.dev

File: docs/de/docs/tutorial/query-params.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+http://127.0.0.1:8000/items/foo?short=on
+```
+
+oder
+
+```
+http://127.0.0.1:8000/items/foo?short=yes
+```
+
+gehen, oder zu irgendeiner anderen Variante der Groß-/Kleinschreibung (Alles groß, Anfangsbuchstabe groß, usw.), dann wird ihre Funktion den Parameter `short` mit dem `bool`-Wert `True` sehen, ansonsten mit dem Wert `False`.
```
Comment (mattkoehne) [inline docs/de/docs/tutorial/query-params.md:129]: "ihre Funktion" -> "Ihre Funktion"

File: docs/de/docs/tutorial/query-params.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+Aber wenn Sie wollen, dass ein Query-Parameter erforderlich ist, vergeben Sie einfach keinen Defaultwert:
+
+```Python hl_lines="6-7"
+{!../../../docs_src/query_params/tutorial005.py!}
+```
+
+Hier ist `needy` ein erforderlicher Query-Parameter vom Typ `str`.
+
+Wenn Sie in ihrem Browser eine URL wie:
```
Comment (mattkoehne) [inline docs/de/docs/tutorial/query-params.md:165]: "ihrem Browser" -> "Ihrem Browser"

**Review by mattkoehne — CHANGES_REQUESTED** [review]
Great work @nilslindemann 👏🏻

File: docs/de/docs/tutorial/query-params.md (hunk 1 above)
Comment (nilslindemann) [reply docs/de/docs/tutorial/query-params.md:129]: Scharfes Auge, hat weder Google Translate, noch LanguageTool, noch ich gesehen.

File: docs/de/docs/tutorial/query-params.md (hunk 2 above)
Comment (nilslindemann) [reply docs/de/docs/tutorial/query-params.md:165]: Dito.

**Comment by nilslindemann** [comment]
Danke, Matthias, fürs Review! Gerne mehr davon :-)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 25eec5b0dc891e21b51bf14262f7acf8e597cd30 at: https://597027dd.fastapitiangolo.pages.dev

**Review by mattkoehne — APPROVED** [review]
Gerne :)

**Comment by alejsdev** [comment]
Thanks for your work! @nilslindemann 🤓💯 

And thanks for the review @mattkoehne 🪄

**Pull request merged**